### DISCUSSION TITLE
Android version cant run on background and cant load skins.

### DISCUSSION THREAD

**Discussion by FransiskusFeroz** [ppy/osu#14559] [Q&A] [answered]
Hi, i just played lazer in mobile and it was not great, cuz its a public alpha. I think that the apk was made with an EXE to APK converter cuz when i leave osu! In the background and open it, IT RESTARTS. And i cant load my homemade skin with [paint.NET](https://www.getpaint.net/)(btw borrowed a computer). The skin is in a .osk btw
Thx! (Btw add taiko and ctb support, if you dont mind)

**Comment by FransiskusFeroz** [comment]
Yep, automatically converted to q&a

**Comment by gagahpangeran** [comment, marked as answer]
> when i leave osu! In the background and open it, IT RESTARTS.

Tracked at https://github.com/ppy/osu-framework/issues/4200

> Btw add taiko and ctb support, if you dont mind

If you mean about mobile input, it's tracked at https://github.com/ppy/osu/issues/4201

All of the issue above is low priority because currently core team member focus to desktop first. Contribution is very welcome for those issue if you or community contributor want to help.

> And i cant load my homemade skin with [paint.NET](https://www.getpaint.net/)(btw borrowed a computer). The skin is in a .osk btw

Please at least provide step by step (or video) about this issue, your skin files, and also log files.

**Reply by sr229** [reply]
I would also like to mention this too:

> I think that the apk was made with an EXE to APK converter 

Keep in mind every build of osu!lazer released is the same version across all platforms, there's no "conversion" used, as Lazer uses Xamarin for the Android/iOS releases.

A nifty guide on cross-platform building should get you acquainted what is going on, as it's impossible to turn an EXE into a full Android package.
